editor/components/block-list/block.js

```diff
@@ -493,14 +509,27 @@ export class BlockListBlock extends Component {
 				] }
 				{ ...wrapperProps }
 			>
+				{ ! isMultiSelected && (
+					<BlockDraggable
+						rootUID={ rootUID }
+						index={ order }
+						uid={ uid }
+						layout={ layout }
+						onDragStart={ this.onDragStart }
+						onDragEnd={ this.onDragEnd }
+						isDragging={ dragging }
+						elementId={ blockElementId }
```

Comment:
The disconnect between this component and the element it clones is unclear to me. At a glance, it seems more sensible to me `<Draggable>` should make its child draggable, not target the target via a DOM ID.

Reply:
The elementId corresponds to the element to clone so it's not exactly the draggable element. Maybe we could rename the prop to clarify it?